ert/data/record/_record.py

```diff
@@ -71,7 +78,21 @@ def record_type(self) -> RecordType:
 
 
 class BlobRecord(Record):
+    """The :class:`BlobRecord` is implementation of the Record class
+    that handles binary data.
```

Comment:
"that treats any stream of bytes as binary data". ? (to defend that we use it for general json strings f.ex).